generate-stats

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
How about calling that "prometheus-stats"? That's the specific [prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/), so it wouldn't hurt to name the file as such and also add this link as a comment?